src/DynamoPackages/PackageManagerSearchElement.cs

```diff
@@ -352,8 +353,8 @@ private List<VersionInformation> TransformVersionsToVersionInformation(Greg.Resp
                 else
                 {
                     // Check within min and max range
-                    isWithinMinMax = (dynamoCompatibility.min == null || dynamoVersion >= Version.Parse(dynamoCompatibility.min)) &&
-                                     (dynamoCompatibility.max == null || dynamoVersion <= Version.Parse(dynamoCompatibility.max));
+                    isWithinMinMax = (dynamoCompatibility.min == null || dynamoVersion >= VersionUtilities.ParseVersionSafely(dynamoCompatibility.min)) &&
```

Comment:
Shouldn't `ParseVersionSafely` be the default `Parse` behavior?

Reply:
Sorry, what do you mean? The problem is that we are sending incomplete versions, like '2019' which fails. If we do 'Version.TryParse' we won't get the version out. So, we need a 'manual' way to get the correct version from an incomplete string version representation.

Reply:
Sorry, I was not clear, by that I meant 2 things:
1. Why even parse "unsafely", this method should replace all instances of `Version.TryParse` or `Version.Parse`?
A few lines [above](https://github.com/DynamoDS/Dynamo/pull/15572/files#diff-a7975ce1cd3828b7bc9a238f4ae9e0e5352d4ab6fba2071899a423ffdf2286ffR305) I can see TryParse being used.
2. Why not name the method as just `VersionUtilities.Parse`?

Reply:
Oh, I see! Good point, cover all the uses, of course. 

I will rename it as proposed, @zeusongit